internal/engine/generate/plpgsql.go

```diff
@@ -265,6 +266,7 @@ func (s *sqlGenerator) VisitExpressionIs(p0 *parse.ExpressionIs) any {
 	if p0.Distinct {
 		str.WriteString("DISTINCT ")
 	}
+	str.WriteString("FROM ")
```

Comment:
Should this be in same block as `str.WriteString("DISTINCT ")` ?

Reply:
Good catch